server/sonar-web/src/main/js/apps/issues/sidebar/CreationDateFacet.js

```diff
@@ -212,6 +212,10 @@ export default class CreationDateFacet extends React.PureComponent {
 
   renderPrefefinedPeriods() {
     const { component, createdInLast, sinceLeakPeriod } = this.props;
+    if (component != null && component.branch != null) {
+      // FIXME handle long-living branches
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1134)